src/main/java/data/duke.txt

```diff
@@ -0,0 +1,4 @@
+T/0/borrow book
```

Comment:
Perhaps it would make sense to exclude files like this being committed to GitHub, since the list of Tasks will always be added at the initialization.